docs/src/pages/getting-started/supported-browsers.md

```diff
@@ -0,0 +1,4 @@
+# Supported Browsers
+
+Material-UI supports the latest, stable releases of all major browsers and platforms.
+On Windows, we support Internet Explorer 11 & Microsoft Edge.
```

Comment:
This makes it sound like those are the only browsers supported on Windows, but I can't think of a better way to phrase it...